initdb.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE seqrdb;
```

Comment:
Is there any way to keep this in the `deploy` folder instead of adding it as a top-level script? Ideally it could live in `/deploy/docker/postgres` or something like that

Reply:
yes!